homeassistant/components/sensor/recorder.py

```diff
@@ -499,10 +499,32 @@ def compile_statistics(  # noqa: C901
 
         result[entity_id]["stat"] = stat
 
+    statistic_ids = statistics.list_statistic_ids(hass, "sum")
```

Comment:
Add a comment to describe the whole section and why we're doing it so future readers understand.